### ISSUE TITLE
GLTFLoader can't load webp images in webworker

### ISSUE THREAD

**Issue by Noobulater** [mrdoob/three.js#29065]
### Description

A GLTFLoader created in a webworker can't load webp files. The issue stems from attempting to use `Image` in a webworker, which doesn't exist. 

### Reproduction steps

1. Run a gtlfloader in a webworker
2. Load a gltf that includes a web file
3. Error message
![image](https://github.com/user-attachments/assets/5e756972-b1e5-41fe-81a5-0bdf5687455d)

### Code

``

### Live example

* [jsfiddle-latest-release WebGLRenderer](https://jsfiddle.net/3mrkqyea/)
* [jsfiddle-dev WebGLRenderer](https://jsfiddle.net/gcqx26jv/)
* [jsfiddle-latest-release WebGPURenderer](https://jsfiddle.net/8L2jkmx7/)
* [jsfiddle-dev WebGPURenderer](https://jsfiddle.net/L3n1w4yh/)

### Version

r167

**Comment by Noobulater** [comment]
Failing in the `detectSupport` method  in [GLTFLoader ](https://github.com/mrdoob/three.js/blob/master/examples/jsm/loaders/GLTFLoader.js)

**Comment by mrdoob** [comment]
But a pnd and/org jpg works?

**Comment by Mugen87** [comment]
@mrdoob Yes, because `detectSupport()` is only used in context of the WebP and AVIF extension. The `detectSupport()` implementations have a dependency to `Image` which is not available in a web worker.

I wonder if `detectSupport()` of `GLTFTextureWebPExtension` and `GLTFTextureAVIFExtension` can be rewritten based on `ImageBitmap` which should solve the issue.

**Comment by Noobulater** [comment]
Also this is an issue with RGBELoader

**Comment by donmccurdy** [comment]
Small PRs fixing issues with using GLTFLoader in Web Workers, Node.js, or Deno are welcome. Some complex adaptations to other runtime environments might be out of scope for three.js, and would need to be handled at the application level, or by forking the loader. I'm happy to review PRs, but unless someone has bandwidth to work on support for these environments (I do not) I'd prefer to close the issue.

**Comment by ashconnell** [comment]
In detectSupport simply resolving true if there is no globalThis.Image allows the GLTFLoader to finish loading in nodejs or webworkers.

Perhaps more importantly, the GLTFLoader doesn't actually bubble this error back up to the loader so in user land it just seems like its loading forever.
